app/src/main/java/com/greenfox/opal/gitinder/service/MockServer.java

```diff
@@ -18,45 +22,90 @@
 import retrofit2.http.Path;
 
 public class MockServer implements ApiService {
-    @Override
-    public MockCall<LoginResponse> login(@Body final LoginRequest loginRequest) {
-        return new MockCall<LoginResponse>() {
-            @Override
-            public void enqueue(Callback<LoginResponse> callback) {
-                LoginResponse response;
-                if (loginRequest.getUsername().isEmpty() || loginRequest.getAccessToken().isEmpty()) {
-                    String message = "Missing parameter(s):";
-                    if (loginRequest.getUsername().isEmpty()) {
-                        message += " username";
-                    }
-                    if (loginRequest.getAccessToken().isEmpty()) {
-                        message += " accessToken";
-                    }
-                    message += "!";
-                    response = new LoginResponse(message);
-                } else {
-                    response = new LoginResponse(loginRequest.getUsername(), loginRequest.getAccessToken());
-                }
-                callback.onResponse(null, Response.success(response));
-            }
-        };
-    }
 
 	@Override
-	public MockCall<Profile> swiping(@Header(value = "X-GiTinder-token") final String token,
-																	 @Path("username") String username,
-																	 @Path("direction") Enum<Direction> direction) {
+	public MockCall<LoginResponse> login(@Body final LoginRequest loginRequest) {
+		return new MockCall<LoginResponse>() {
+			@Override
+			public void enqueue(Callback<LoginResponse> callback) {
+				LoginResponse response;
+				if (loginRequest.getUsername().isEmpty() || loginRequest.getAccessToken().isEmpty()) {
+					String message = "Missing parameter(s):";
+					if (loginRequest.getUsername().isEmpty()) {
+						message += " username";
+					}
+					if (loginRequest.getAccessToken().isEmpty()) {
+						message += " accessToken";
+					}
+					message += "!";
+					response = new LoginResponse(message);
+				} else {
+					response = new LoginResponse(loginRequest.getUsername(), loginRequest.getAccessToken());
+				}
+				callback.onResponse(null, Response.success(response));
+			}
+		};
+	}
```

Comment:
@dorinagy this methods indentation is different from the others, and also it uses tabs instead of spaces